src/lightbox/Lightbox.js

```diff
@@ -59,6 +59,11 @@ export default function Lightbox(props: Props) {
       ? `Shared in #${streamNameOfStreamMessage(message)}`
       : 'Shared with you';
   const resource = getResource(src, auth);
+
+  // Since we're using `Dimensions.get` (below), we'll want a rerender
+  // when the orientation changes. No need to store the value.
+  useSelector(state => getSession(state).orientation);
```

Comment:
Oh, I think I see this as a bug in the existing implementation too!

Start in portrait mode, open the lightbox, and then rotate to landscape. The image stays limited to a pillarbox, even if the image's natural aspect ratio is wider. And the lightbox header and footer stay narrow, too:
![image](https://user-images.githubusercontent.com/28173/106219199-c34fc500-618d-11eb-8cd9-750a735944b5.png)

I'd noticed that a few minutes ago, and was going to mention it as another small design issue that'd be good to fix while we're working on this -- the header and footer should go the whole width of the screen. Then I realized that it's caused by the bug this change fixes :slightly_smiling_face: 

Not sure if I'd ever spotted that before; if I had, I must have put it down to a suboptimal design choice then too.

Conversely, if you start in landscape and rotate to portrait, everything stays wide, and the header and footer are unusable because the controls are off-screen. That must be this same bug. (And, even before seeing this code, would be a signal that it's an outright bug and not a dubious design choice.)

Reply:
There's something else that looks wrong in that screenshot: there's too much blank dark space at the *top* of the footer, and at the *bottom* of the header. I'm not seeing that; is it possible you had made a code adjustment that was causing that? Or perhaps it's this same bug (though I don't really see how)?